wp-includes/class-wp-service-workers.php

```diff
@@ -78,31 +78,19 @@ public function get_registry() {
 	/**
 	 * Get the current scope for the service worker request.
 	 *
-	 * @global WP $wp
-	 *
-	 * @return int Scope. Either SCOPE_FRONT, SCOPE_ADMIN, or if neither then 0.
+	 * @todo We don't really need this. A simple call to is_admin() is all that is required.
```

Comment:
@westonruter Not related to this specifically but a little bit related in terms of checking for `self::QUERY_VAR`:
Should we only `init` `WP_Service_Worker_Scripts` if the `self::QUERY_VAR` does exist? Otherwise wouldn't the scripts always be registered but only served if `self::QUERY_VAR` is set? Wouldn't the scripts need to be registered only in case of serving the service worker? Thoughts?